web/app/src/Layout/PlayingCards.js

```diff
@@ -0,0 +1,99 @@
+import './PlayingCards.css'
+import React, {Component} from "react";
+
+class PlayingCards extends Component {
+    state = {
+	tasks: [
+      {id: "1", taskName:"Trample 1",type:"cardsInHand", backgroundImage: "red"},
```

Comment:
We will have different cards. In the future, we will be getting the cards from python classes. So, for now I feel this will be fine till our UI pings the midtier.